app/views/pieces/show.html.haml

```diff
@@ -3,10 +3,11 @@
   %h5.show-composer= @piece.composer
 
 - if user_signed_in?
-  - if current_user == @piece.user || current_user.role == "admin"
+  - if current_user == @piece.user || current_user.admin
     .links
```

Comment:
Another use for an `editable_by?` method on your piece - this logic is duplicated across several places